core/register_core_types.cpp

```diff
@@ -155,6 +158,9 @@ void register_core_types() {
 	ClassDB::register_class<StreamPeerTCP>();
 	ClassDB::register_class<TCP_Server>();
 	ClassDB::register_class<PacketPeerUDP>();
+	ClassDB::register_class<UDPServer>();
```

Comment:
We have `TCP_Server` and `UDPServer` using different naming conventions. The former should be fixed, breaking compat in 4.0, I guess?

Reply:
> The former should be fixed, breaking compat in 4.0, I guess?

Yeah, I agree, `TCP_Server` should become `TCPServer`